lib/plugin/metadata.rb

```diff
@@ -27,6 +27,7 @@ class Plugin::Metadata
     "discourse-category-experts",
     "discourse-characters-required",
     "discourse-chat-integration",
+    "discourse-chat-integration",
```

Comment:
```suggestion
    "discourse-chat",
    "discourse-chat-integration",
```

Reply:
🤦 derp

Reply:
typo in the branch name too... lookin' good over here